DREAM/tasdreamExternalTests.cpp

```diff
@@ -278,7 +273,7 @@ bool ExternalTester::testGamma1D(){
 }
 
 bool ExternalTester::testGaussian2D(){
-    int s = (rngseed == -1) ? 12 : rngseed;
+    int s = (rngseed == -1) ? 25 : rngseed;
```

Comment:
Did you mean to change that in this PR??

Reply:
I think this was related to windows testing and the way windows flags are handled, 12 was failing on windows because of differences in the new build system.

Reply:
Actually, this was related to the build flags in the simple GNU Make build engine. Still, this is part of the build engine modifications.